### ISSUE TITLE
Accessibility Issue - fixed font size in css styles.

### ISSUE THREAD

**Issue by Roomka** [learningequality/ka-lite#3352]
Branch:master
Expected Behavior: When changing font size from browser the page/site should adjust the font correctly.
WCAG Guidelines: 1.4.3 Resize text (Level AA)
Current Behavior: The text remains the same size after changing font preferences in browser.
Steps to reproduce in chrome (similar to other browsers): 
1.Go to settings -> advanced settings
2.Change font size to large/small
3.Check a page from the server

![font_size1](https://cloud.githubusercontent.com/assets/4850405/6743218/7b445fda-ce97-11e4-95a9-68290a612e13.png)
![font_size2](https://cloud.githubusercontent.com/assets/4850405/6743217/7b44600c-ce97-11e4-8683-8ce85ce99665.png)

**Comment by MCGallaspy** [comment]
Looks like 0.12.x, but someone can check if the issue exist on 0.13.x & develop.

**Comment by radinamatic** [comment]
I don't have 0.13 on this machine to test, but on Develop the font size seem determined by the bootstrap.min.css, and I doubt it makes any sense to try to fix that. I tested zoom options in FF, Chrome & IE: Page zoom works in all and KA Lite behaves well up to 400% zoom (media-queries firing fine). What does not work (as stated above) is the Text-only zoom on Chrome & IE (on FF it does).

I will recheck with the Zoom Text (another program used by people with low vision), but it works differently and does not depend on font size setup for browsers.

All in all, I would say that this requires neither urgent nor essential fix.

**Comment by shreab373** [comment]
Hello, I'm new, i would like to work on this, could you guide me as to what needs to be done?

**Comment by benjaoming** [comment]
Hi @shreab373 - welcome :)

Can definitely help you get this issue solved :)

The first place to go is our developer docs: http://ka-lite.readthedocs.io/en/develop/developer_docs/index.html

From there on, get a working environment set up..

Can I ask curiously why this issue exactly is something you want or need to have solved?

**Comment by shreab373** [comment]
hello, I'm a beginner to open source contributions, i was looking at easier things to fix, in your opinion is this a good first bug?

**Comment by benjaoming** [comment]
@shreab373 - depends, do you know Python? CSS?

**Comment by shreab373** [comment]
i do know python and but i'm only familiar with css as i had used in a project which was about a year ago..

**Comment by benjaoming** [comment]
Well, the issue is probably something that will require you to do some research on CSS and font handling. But you also have to research how to get the development environment for KA Lite running ;) Once you have it up, it's probably easier for you to understand how to make the necessary changes and if you want to work on this issue in particular.

It requires a simple fix, but it's depending on whether you want to research it :)

**Comment by shreab373** [comment]
i shall first set up dev env and get back to this as soon as that is done! thanks :)

**Comment by radinamatic** [comment]
@shreab373 Check these WCAG resources:

[1.4.4 Resize text (Level AA)](https://www.w3.org/WAI/WCAG20/quickref/#qr-visual-audio-contrast-scale)
[C28: Specifying the size of text containers using em units](https://www.w3.org/TR/WCAG20-TECHS/C28.html)
[Font Size on the Web](http://accessibility.psu.edu/fontsizehtml/)
